app/src/main/java/acr/browser/lightning/adblock/AbpListUpdater.kt

```diff
@@ -140,6 +140,12 @@ class AbpListUpdater @Inject constructor(val context: Context) {
                 abpDao.update(entity)
                 return false
             }
+            if (response.code == 404) {
```

Comment:
Should we instead check if the code is not 200, to handle all errors?

Reply:
I thought about this, but I don't know much about the http status codes, and wasn't sure whether there are more than `200` that should be accepted.
But changing to `if (response.code != 200)` and adapting the message should work too.

Reply:
I had a look at https://developer.mozilla.org/en-US/docs/Web/HTTP/Status.
Maybe we could also accept 203, and handle 301, 307 and 308.

Reply:
I would not bother. I reckon 200 should do just fine in practice. Besides all the redirects cases should be handled internally by the framework doing the download really.